openslides_backend/action/actions/poll/update.py

```diff
@@ -114,6 +116,17 @@ def update_instance(self, instance: dict[str, Any]) -> dict[str, Any]:
                 "Following options are not allowed in this state and type: "
                 + ", ".join(not_allowed)
             )
+
+        # check named and live_voting_enabled
+        if instance.get("live_voting_enabled") and not (
+            poll["type"] == Poll.TYPE_NAMED
+            and poll.get("content_object_id")
+            and collection_from_fqid(poll["content_object_id"]) == "motion"
```

Comment:
`content_object_id` is a required field, therefore checking if `poll.get("content_object_id")` should be unnecessary